README.md

```diff
@@ -4,18 +4,30 @@
-2. If you need or want to store specific files with your idea (such as diagrams), create a new branch within the brainBox repo. Name it: `[project shortname]-[your name]`. For example: `choco-factory-charlie`. Add the new directory under the `brainy-ideas` directory. The project shortname that you use for the GitHub issue, the shortname that you use for the new branch, and the shortname that you use for the directory within the repo should be identical.
-3. When you are ready, move the issue into the "Roasting" pipeline; open your idea up for group discussion by tagging the lab.
-4. Watch and respond to the questions that come in. Feel free to remind certain individuals to comment with their thoughts (by tagging them individually) if you feel they would have particularly valuable expertise to hone your idea.
-5. As your idea takes full shape, move the issue into the "Brewing" pipeline. Craft a clear pitch for the PI's approval.
-6. Use Calendly to pre-set a meeting with the PI, at least two days after you intend to submit the pull request on GitHub.
-7. Submit the pull request on GitHub by adjusting the details on the default pull request form.
-8. Get excited to discuss how to make your idea a reality!
+
+**Roasting**
+
+2. Create a new branch within the brainBox repo. Name it: `[project shortname]-[your name]`. For example: `choco-factory-charlie`. Add a new directory (subfolder) under the `brainy-ideas` directory. The project shortname that you use for the GitHub issue, the shortname that you use for the new branch, and the shortname that you use for the sub-directory should be identical.
+3. On your branch, within your project subfolder in the `brainy-ideas` directory, create a readme file. Use this file to build out and refine your idea.  This can start as a duplication of the content of your issue. 
+4. When you are ready, move the issue into the "Roasting" pipeline; open your idea up for group discussion by tagging the lab (@labwide) on the issue.
+5. Watch and respond to the questions that come in. Feel free to remind certain individuals to comment with their thoughts (by tagging them individually) if you feel they would have particularly valuable expertise to hone your idea.
+6. As you refine the idea, update the readme in your subdirectory of `brainy-ideas` on your own, speciifc branch. Think of the issue as the conversation and the readme as the work-in-progress.
+
+**Brewing**
+
+7. As your idea takes full shape, move the issue into the "Brewing" pipeline. Craft a clear pitch for the PI's approval using the pull request template.
+8. Use [Calendly](https://calendly.com/gbuzzell/30-minute-meeting) to pre-set a meeting with the PI, at least two days after you intend to submit the pull request on GitHub.
```

Comment:
@jessb0t Not sure about this? I do LOVE the use of calendly for scheduling. And, I agree a meeting is likely useful at some point. But, I think I prefer to make the standard be that, at least for the first PR, no meeting is set. I can always request one, but I think it might not be best to make everyone that wants to pitch schedule a meeting. Does that make sense?

delete number 8, and add another sentence to number 7 saying something like "It is not necessary to schedule a meeting with the PI to discuss the PR, however, if you think it will help, then  use [Calendly](https://calendly.com/gbuzzell/30-minute-meeting) to pre-set a meeting with the PI, at least two days after you intend to submit the pull request on GitHub.